testing/ncids-js-testing/src/auto-init.ts

```diff
@@ -0,0 +1,4 @@
+import './index.scss';
+
+import { NCIExtendedHeaderWithMegaMenuAutoInit } from '@nciocpl/ncids-js';
+new NCIExtendedHeaderWithMegaMenuAutoInit();
```

Comment:
The idea with the autoinit is that you do not have to execute it. The import should be all you need.